### DISCUSSION TITLE
direct audio import?

### DISCUSSION THREAD

**Discussion by IAmAThousandTrees** [ManimCommunity/manim-voiceover#91] [Ideas]
I'm wondering if it would be a good idea and easy to implement the facility to simply specify an audio file (preferably wav not mp3) for each spoken line? then a basic recorder with effects like audacity could be used to record the entire script, apply needed effects like noise-filter and compression, then cut it into segments that manim_voiceover could use for it's timings and render into the finished file?

**Comment by r-debashis** [comment]
I think that will be a great feature, considering the fact that recording voice multiple times can be painful. Specially if you want to keep each take error-free, and having the exact same time duration.
Having the ability to import recorded voice over from mp3/wav files would reduce a lot of pain.
